pkg/steps/pod_test.go

```diff
@@ -14,6 +15,13 @@ import (
 	"github.com/openshift/ci-tools/pkg/api"
 )
 
+const (
+	buildID      = "test-build-id"
+	jobName      = "very-cool-prow-job"
+	pjID         = "prow-job-id"
+	jobSpecValue = `{"type":"periodic","job":"periodic-ci-azure-e2e-applysecurityupdates","buildid":"21","prowjobid":"ec28bec2-b7a4-11e9-af8e-0a58ac108dbc","extra_refs":[{"org":"openshift","repo":"openshift-azure","base_ref":"master"}]}`
```

Comment:
Might as well use a presubmit or batch config as well to test all the env vars we don't set with periodics

Reply:
Will do today.
I was keeping things as simple as possible yesterday, because, you know, the sorting thing was eating me alive.

Reply:
Done.